spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  let(:suppervisor){FactoryBot.create :user, name: "Name 2", email: "test123456789@mail.com", role: "suppervisor"}
+  let(:trainee){FactoryBot.create :user, name: "Name 2", email: "test12345678910@mail.com", role: "trainee"}
+
+
+  before{login_as suppervisor}
+
+  describe "GET /index" do
+    it "render template index" do
+      get :index
+      expect(response).to render_template :index
+    end
+  end
+
+  describe "GET new" do
+    before{get :new}
+
+    it "render template new" do
```

Comment:
```suggestion
    it "should render template new" do
```
Ở chỗ mô tả test case em thêm từ should vào nhé

Reply:
vâng ạ